usaspending_api/api_docs/api_documentation/download/README.md

```diff
@@ -0,0 +1,40 @@
+# Download
+
+The following endpoints are involved with generating files that reflect the site's underlying data. 
+
+## Award Data Archive
+
+On a monthly basis, the website pre-generates a series of commonly used files based on the agency, fiscal year, and award type. These can be found at [Award Data Archive](https://beta.usaspending.gov/#/bulk_download/award_data_archive), or, if you prefer, you can also use the API's [List Downloads Endpoint](https://github.com/fedspendingtransparency/data-act-documentation/blob/master/usaspending/api-documentation/download/List%20Downloads.md).
```

Comment:
These links are broken. The Award Data Archive link was updated to https://beta.usaspending.gov/#/download_center/award_data_archive. We also need to remove the beta prefix for launch.

Reply:
nice!